deploy/00_deployMocks.ts

```diff
@@ -1,38 +1,37 @@
-import { HardhatRuntimeEnvironment } from "hardhat/types";
-import { DeployFunction } from "hardhat-deploy/types";
-import "hardhat-ethernal";
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { DeployFunction } from 'hardhat-deploy/types';
+import 'hardhat-ethernal';
 
 const deployMocks: DeployFunction = async function (
   hre: HardhatRuntimeEnvironment,
 ) {
   const { deployments, getNamedAccounts, getChainId } = hre;
-  const { deploy } = deployments;
+  const { deploy, log } = deployments;
   const { deployer } = await getNamedAccounts();
   const chainId = await getChainId();
-
-  if (chainId === "31337") {
-    console.log(`Deploying chainlink mocks to chainId: ${chainId}`);
-    const linkToken = await deploy("LinkToken", {
+  if (chainId === '31337') {
+    log(`Deploying chainlink mocks to chainId: ${chainId}`);
```

Comment:
No idea where this logs it? Kinda useless.

Reply:
`const { deploy, log } = deployments;`
but I didn't touch this as well

Reply:
again, check the code, it was changed, I know where the `log` is from, I just don't know where it logs haha.
Not sure if you know, if not, we should just use `console.log` I guess.